andela/AndelaEatsUI
# #160818697 Implement add menu modal UI

**What does the PR do?**
- Designs `Add Menu UI`

**Description of Task to be completed?**
- add `Add menu modal`
- write test for add menu modal

**How should this be manually tested?**
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-implement-add-menus-UI-160818697`
- run these commands `npm run bundle` and `npm run dev` 
- Log in as an `Admin`, navigate to `admin dashboard`, select `Menu Icon`, and click on `Add Menu`
- run the command `npm run test-dev` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#160818697](https://www.pivotaltracker.com/story/show/160818697)

**Screenshots**
![screen shot 2018-10-10 at 3 08 54 pm](https://user-images.githubusercontent.com/26305570/46742424-e7c2c300-cc9e-11e8-8388-dcae8218f879.png)
![screen shot 2018-10-10 at 3 13 21 pm](https://user-images.githubusercontent.com/26305570/46742523-1d67ac00-cc9f-11e8-958e-1a85104187fe.png)
